ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -317,6 +536,86 @@ export default {
       netRegexJa: NetRegexes.startsUsing({ source: 'スティギモロク・ロード', id: '57D8', capture: false }),
       response: Responses.getOut(),
     },
+    {
+      id: 'DelubrumSav Queen Cleansing Slash',
+      // PLD and GNB tank invulnerabilities do not get Physical Vulnerability Up
+      // Tank swap will be required between the two hits if not using a tank invulnerability
+      // Tank swap required after second hit if not using PLD or GNB tank invulnerabilities
+      // To avoid bad swaps between 11 other tanks, only mention swap to targetted tank
+      netRegex: NetRegexes.startsUsing({ source: 'The Queen', id: '59F5' }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Kriegsgöttin', id: '59F5' }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Garde-La-Reine', id: '59F5' }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'セイブ・ザ・クイーン', id: '59F5' }),
+      condition: tankBusterOnParty,
+      response: (data, matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          tankBusterAndSwap: {
+            en: 'Tank Buster + Swap',
+            de: 'Tankbuster + Wechsel',
+            fr: 'Tank buster + Swap',
+            ja: 'タンクバスター + スイッチ',
+            cn: '死刑 + 换T',
+            ko: '탱버 + 교대',
+          },
+          tankBusters: Outputs.tankBusters,
+          tankBusterOnYou: Outputs.tankBusterOnYou,
+          tankInvuln: {
+            en: 'Invuln Tank Buster',
+          },
+        };
+
+        if (data.me === matches.target) {
+          if (data.role === 'tank') {
+            if (data.job === 'PLD' || data.job === 'GNB')
+              return { alertText: output.tankInvuln() };
+            return { alertText: output.tankBusterAndSwap() };
+          }
+          return { alarmText: output.tankBusterOnYou() };
+        }
+        if (data.role === 'healer')
+          return { alertText: output.tankBusters() };
```

Comment:
```suggestion
        if (data.role === 'healer' || data.role === 'tank')
          return { alertText: output.tankBusterOnPlayer({ player: matches.target });
```

I think it's worth calling out the target, and informing tanks about it, as well.

Reply:
The condition would need to be modified as well to inform the healers/tanks outside of the party.

Reply:
Sorry, I removed the party condition after the comment in 04c9ea76b46e3bd73f8a85a9e5bf90b64c4bd9fa